src/components/card/Card.stories.tsx

```diff
@@ -265,7 +256,7 @@ export const cardTest = (): React.ReactElement => (
       <CardMedia imageClass="add-aspect-16x9">
         {/* eslint-disable-next-line jsx-a11y/img-redundant-alt */}
         <img
-          src="https://images.unsplash.com/photo-1583974098224-f64f67f46778?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=800&h=800&fit=crop&ixid=eyJhcHBfaWQiOjF9"
+          src={sparklerImage}
```

Comment:
Not sure if we care, but some of these images are not the original images (noticed it in Happo):
![image](https://user-images.githubusercontent.com/15805554/119885645-15b4ae80-bf00-11eb-8861-792bde59ba85.png)

Reply:
Yeah, I should have mentioned that! Some of the images seemed to be only used a few times and I didn't see a reason why we couldn't pare down our list.

If I'm wrong, I'm happy to change it